### PR TITLE
cuTENSOR v1.3.0

### PR DESCRIPTION
**Pull request by takagi**

### CODE REVIEW

**Comment by takagi** [comment]
pfnCI, test this please.

**Comment by takagi** [comment]
pfnCI, test this please.

**Comment by takagi** [comment]
Rebased on #654.

File: docker.py [hunk 1]
```diff
@@ -867,6 +869,10 @@ def get_scipy_choices():
 codes['cutensor1.2.0-cuda110'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.2.0.2 1.2.0 11.0;'
 codes['cutensor1.2.1-cuda111'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.2.1.7 1.2.1 11.1;'
 codes['cutensor1.2.2-cuda112'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.2.2.5 1.2.2 11.2;'
+codes['cutensor1.3.0-cuda102'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.3.0.3 1.3.0 10.2;'
+codes['cutensor1.3.0-cuda110'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.3.0.3 1.3.0 11.0;'
+codes['cutensor1.3.0-cuda111'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.3.0.3 1.3.0 11.1;'
+codes['cutensor1.3.0-cuda112'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.3.0.3 1.3.0 11.2;'
```
Comment (kmaehashi) [inline docker.py:875]:
```suggestion
codes['cutensor1.2.2-cuda112'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.2.2.5 1.2.2 11;'
codes['cutensor1.3.0-cuda102'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.3.0.3 1.3.0 10.2;'
codes['cutensor1.3.0-cuda110'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.3.0.3 1.3.0 11.0;'
codes['cutensor1.3.0-cuda111'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.3.0.3 1.3.0 11;'
codes['cutensor1.3.0-cuda112'] = 'RUN eval $CUTENSOR_INSTALL && install_cutensor 1.3.0.3 1.3.0 11;'
```

**Pull request merged**